package.json

```diff
@@ -4,6 +4,11 @@
   "description": "Fast and low overhead web framework, for Node.js",
   "main": "fastify.js",
   "types": "fastify.d.ts",
+  "type": "commonjs",
+  "exports": {
+    "import": "./fastify.mjs",
+    "require": "./fastify.js"
+  },
```

Comment:
please add `package.json` as well, otherwise fastify-plugin would not work. Add also https://github.com/fastify/fastify/blob/master/lib/symbols.js

Reply:
Can you elaborate? I am not getting this one, sorry :(

Reply:
He's saying that this needs to export `package.json` as an entity from the module.

Reply:
OK. I think we need to track what is considered the public API and add all of the files in the exports.

Reply:
That's what I'm saying.

Reply:
Ok, now I got it. This is the only public API, am I correct?

Reply:
Also symbols, I do not remember which module used them, but I think one did.